packages/docs/src/patterns/footer-wrapper/examples/footer-wrapper-slot.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+	<FooterWrapper>
+		<VRow>
+			<VSpacer />
+			<p>
```

Comment:
```suggestion
		<VRow>
			<VSpacer />

			<p class="mb-0>
```
Je pense que ça doit faire un décalage actuellement